apps/src/templates/instructions/TopInstructionsHeader.jsx

```diff
@@ -10,72 +10,6 @@ import i18n from '@cdo/locale';
 import color from '../../util/color';
 import styleConstants from '../../styleConstants';
 
-const styles = {
```

Comment:
I realized this wasn't caught because we're using a functional component here, I just moved it to the bottom.